### PR TITLE
CreateVerifyIDIntensityContaminationMetricsFile

### PR DESCRIPTION
**Pull request by gbggrant**
### Description

CreateVerifyIDIntensityContaminationMetricsFile
A simple  program to create a standard picard metrics file
from the output of VerifyIDIntensity.

----

### Checklist (never delete this)

Never delete this, it is our record that procedure was followed. If you find that for whatever reason one of the checklist points doesn't apply to your PR, you can leave it unchecked but please add an explanation below.

#### Content
- [ ] Added or modified tests to cover changes and any new functionality
- [ ] Edited the README / documentation (if applicable)
- [ ] All tests passing on Travis

#### Review
- [ ] Final thumbs-up from reviewer
- [ ] Rebase, squash and reword as applicable

For more detailed guidelines, see https://github.com/broadinstitute/picard/wiki/Guidelines-for-pull-requests

### CODE REVIEW

File: src/main/java/picard/arrays/CreateVerifyIDIntensityContaminationMetricsFile.java [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+    protected int doWork() {
+        IOUtil.assertFileIsReadable(INPUT);
+        final File metricsFile = new File(OUTPUT + "." + FILE_EXTENSION);
+        IOUtil.assertFileIsWritable(metricsFile);
+
+        final MetricsFile<VerifyIDIntensityContaminationMetrics,?> verifyIDIntensityContaminationMetricsFile = getMetricsFile();
+
+        final Pattern HEADER_PATTERN = Pattern.compile("^ID\\s+%Mix\\s+LLK\\s+LLK0\\s*$");
+        final Pattern DASHES_PATTERN = Pattern.compile("^[-]+$");
+        final Pattern DATA_PATTERN = Pattern.compile("^(\\d+)\\s+(\\d*\\.\\d+)\\s+(\\d+)\\s+(\\d+)\\s*$");
```
Comment (pshapiro4broad) [inline src/main/java/picard/arrays/CreateVerifyIDIntensityContaminationMetricsFile.java]: Exponential notation not required for the double field?

File: src/main/java/picard/arrays/CreateVerifyIDIntensityContaminationMetricsFile.java [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+        try(BufferedReader br = new BufferedReader(new FileReader(INPUT))) {
```
Comment (pshapiro4broad) [inline src/main/java/picard/arrays/CreateVerifyIDIntensityContaminationMetricsFile.java]: whitespace
```suggestion
        try (BufferedReader br = new BufferedReader(new FileReader(INPUT))) {
```

File: src/main/java/picard/arrays/CreateVerifyIDIntensityContaminationMetricsFile.java [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+
+    public static final String FILE_EXTENSION = "verifyidintensity_metrics";
+
+    @Override
+    protected int doWork() {
+        IOUtil.assertFileIsReadable(INPUT);
+        final File metricsFile = new File(OUTPUT + "." + FILE_EXTENSION);
+        IOUtil.assertFileIsWritable(metricsFile);
+
+        final MetricsFile<VerifyIDIntensityContaminationMetrics,?> verifyIDIntensityContaminationMetricsFile = getMetricsFile();
```
Comment (pshapiro4broad) [inline src/main/java/picard/arrays/CreateVerifyIDIntensityContaminationMetricsFile.java]: whitespace
```suggestion
        final MetricsFile<VerifyIDIntensityContaminationMetrics, ?> verifyIDIntensityContaminationMetricsFile = getMetricsFile();
```

File: src/main/java/picard/arrays/CreateVerifyIDIntensityContaminationMetricsFile.java (hunk 1 above)
Comment (gbggrant) [reply src/main/java/picard/arrays/CreateVerifyIDIntensityContaminationMetricsFile.java]: Not that I've seen.

**Comment by gbggrant** [comment]
retest this please

**Pull request merged**